pygenesys/version.py

```diff
@@ -64,5 +64,5 @@
 MICRO = _version_micro
 VERSION = __version__
 PACKAGE_DATA = {'pygenesys': [pjoin('data', '*')]}
-REQUIRES = ["numpy"]
+REQUIRES = ['numpy', 'pandas', 'matplotlib', 'pytest']
```

Comment:
At some point you may have to specify specific versions of your dependencies that you support. You may want to add a note to your readme describing the versions of each that you're using to develop to be transparent.